packages/cache/RELEASES.md

```diff
@@ -49,4 +49,7 @@
   - Use @azure/storage-blob v12.8.0
 
 ### 1.0.10
-- Update `lockfileVersion` to `v2` in `package-lock.json [#1022](https://github.com/actions/toolkit/pull/1022) 
\ No newline at end of file
+- Update `lockfileVersion` to `v2` in `package-lock.json [#1022](https://github.com/actions/toolkit/pull/1022) 
+
+### 1.0.11
+- Added support to check if Artifact cache service feature is available or not.
```

Comment:
please link the PR to this line for future reference

Reply:
done